README.md

```diff
@@ -88,7 +88,7 @@ bundle install
 - Run tests
 
 ```sh
-bundle exec rspec
+bundle exec bin/rspec
```

Comment:
Happy to revert this if the old command works on your machines but on mine I absolutely have to specify the `bin/`